tmuxer

```diff
@@ -2,6 +2,13 @@
 
 set -euo pipefail
 
+MAJOR_BASH_VERSION=$(echo $BASH_VERSION | cut -d '.' -f 1)
+if [[ $MAJOR_BASH_VERSION < 4 ]]
+then
+  echo "You must use a Bash shell that is at least version 4 (you're on $BASH_VERSION)" >&2
+  exit 1
```

Comment:
Can this be updated to use the `die` function that's used elsewhere for logging + exiting?

Reply:
Sure thing. It's been updated now.